Consistent file naming across architectures for so libs and cmake files:
remove intermediate suffix like ".cpython-36m-x86_64-linux-gnu".
Avoid overwriting the "real" cmake file with the generic wrapper's one (same
name in such a case)

--- a/sources/shiboken2/CMakeLists.txt
+++ b/sources/shiboken2/CMakeLists.txt
@@ -98,6 +98,11 @@
         "PYTHON_CONFIG_SUFFIX is empty. It should never be empty. Please file a bug report.")
 endif()
 
+# consistent file naming across architectures
+set(PYTHON_EXTENSION_SUFFIX "")
+set(PYTHON_CONFIG_SUFFIX "")
+set(PYTHON_SHARED_LIBRARY_SUFFIX "")
+
 message(STATUS "PYTHON_EXTENSION_SUFFIX:      ${PYTHON_EXTENSION_SUFFIX}")
 message(STATUS "PYTHON_CONFIG_SUFFIX:         ${PYTHON_CONFIG_SUFFIX}")
 message(STATUS "PYTHON_SHARED_LIBRARY_SUFFIX: ${PYTHON_SHARED_LIBRARY_SUFFIX}")
--- a/sources/shiboken2/data/CMakeLists.txt
+++ b/sources/shiboken2/data/CMakeLists.txt
@@ -37,16 +37,11 @@
      PATH_VARS SHIBOKEN_PYTHON_MODULE_DIR SHIBOKEN_SHARED_LIBRARY_DIR
 )
 
-configure_file("${CMAKE_CURRENT_SOURCE_DIR}/Shiboken2Config.cmake.in"
-               "${CMAKE_CURRENT_BINARY_DIR}/Shiboken2Config.cmake" @ONLY)
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/Shiboken2ConfigVersion.cmake.in"
                "${CMAKE_CURRENT_BINARY_DIR}/Shiboken2ConfigVersion.cmake" @ONLY)
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/shiboken2.pc.in"
                "${CMAKE_CURRENT_BINARY_DIR}/shiboken2${shiboken2_SUFFIX}.pc" @ONLY)
 
-install(FILES "${CMAKE_CURRENT_BINARY_DIR}/Shiboken2Config.cmake"
-        DESTINATION "${LIB_INSTALL_DIR}/cmake/Shiboken2-${shiboken2_VERSION}")
-
 install(FILES "${CMAKE_CURRENT_BINARY_DIR}/install/Shiboken2Config${PYTHON_CONFIG_SUFFIX}.cmake"
         DESTINATION "${LIB_INSTALL_DIR}/cmake/Shiboken2-${shiboken2_VERSION}")
 
